src/drivers/net/intel.h

```diff
@@ -200,6 +200,14 @@ struct intel_descriptor {
 #define INTEL_FEXTNVM11 0x05bbcUL
 #define INTEL_FEXTNVM11_WTF	0x00002000UL	/**< Don't ask */
 
+/** I210 Packet Buffer Sizes */
+#define INTEL_E1000_RXPBS	0x02404
+#define INTEL_E1000_TXPBS	0x03404
+#define INTEL_I210_DTXMXPKTSZ	0x355C
```

Comment:
Same comment: `DTXMXPKTSZ` exists for some other E1000 variants, so let's just use `INTEL_DTXMXPKTSZ`.